docs/replication-retry.md

```diff
@@ -0,0 +1,134 @@
+# Replication Retry
+
+## Description
+
+This feature offers a way for users to monitor and retry failed replication
+operations. It enables users to retrieve a list of failed operations and retry
+specific replication operations.
+
+## Requirements
+
+* The listing of failed operations.
+* The retry of a specific failed operation.
+
+## Design
+
+A RESTful API will expose methods for users to list and retry failed operations.
+See [Explanations](#explanations) for more detail.
+
+## Definition of API
+
+* GET `/_/crr/list/failed/all`
```

Comment:
maybe `/list` is superfluous, i.e. `/_/crr/failed/...` can be simpler and organized enough for the purpose.

Reply:
The idea behind is that a GET already carries the notion of retrieval, so if the GET occurs on a "directory-like" structure it implies we want a listing of this structure.

Reply:
So for getting all failed entries, you could also support directly `GET /_/crr/failed`

Reply:
Good point—this simplifies the route. I will update this.